packages/inventory-general-info/package.json

```diff
@@ -3,12 +3,13 @@
     "version": "1.0.5",
     "description": "System info detail for RedHat Cloud Services project.",
     "main": "index.js",
```

Comment:
^^

Reply:
Oh now it hit me. We might even need the "files" property of you want to publish more that cjs/index and esm/index. That would be probably for the umd build which is not listed anywhere and might get ignored.

Reply:
The `files` field is just alternative to `.npmignore` https://docs.npmjs.com/files/package.json#files they work in different way

> Files included with the “package.json#files” field cannot be excluded through .npmignore or .gitignore.